tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 envlist =
-    py{35,36,37}-lint
-    py{35,36,37}-install-geth-{
+    py{36,37,38,39}-lint
```

Comment:
If we're dropping `36` below, should we also drop it for the lint runs?

Reply:
I'm good to drop it. It seemed like we used to keep linting for older versions but it doesn't seem to do too much as far as verifying support anyhow so I will drop it and merge 👍

Reply:
on second thought, I guess we just kept the py34 lint profile but never triggered it here... will drop py36! We do know it works at least for the time being too so this will just keep it down to the 3 latest versions as before.